packages/core/pluggableElementTypes/models/BaseTrackModel.ts

```diff
@@ -53,7 +53,8 @@ export function createBaseTrackModel(
       /**
        * #property
        */
-      configuration: ConfigurationReference(baseTrackConfig),
+      // @ts-ignore
```

Comment:
not exactly sure how to remove ts-ignore here, should be that AnyConfigurationSchemaType is a IAnyModelType i think but can't convince it of that

Reply:
can maybe just cast instead of ts-ignore?

Reply:
would have to cast "baseTrackConfig as unknown as IAnyModelType" which is probably not better. probably the best action would need to look at configurationSchema.ts to see why it doesn't satisfy IAnyModelType